### ISSUE TITLE
Discrepancy between 6B and 627B version

### ISSUE THREAD

**Issue by punisher1k** [Olivia-fsm/DoGE#1]
Hi @Olivia-fsm, thank you for sharing your great work.

I am running your provided code on the 6B version of the SlimPajama data set and obtained <= 7. ppl across all domains. I am wondering if this is an unexpected behavior (your reported ppl for C4 and CC are >40).

Besides, could you please share your *bin files for the 627B SlimPajama? The processing time for the dataset is too long is my machine so it would be great if you could also share them with me.

Thank you in advance.

<img width="1085" alt="image" src="https://github.com/Olivia-fsm/DoGE/assets/110251652/119728d1-17c2-4452-bf3a-783012cc26db">

**Comment by sowmaster** [comment]
Hi, were you able to run this code as provided or did you have to make few changes/debugging? Also did you run it on a single or multi GPUs? 
Thanks!

**Comment by sowmaster** [comment]
BTW I think your plots show the log perplexities and not the actual perplexities which would be why you have smaller values.

**Comment by punisher1k** [comment]
Yes, I reproduced the exp using the exact config but my obtained Github domain weight is often large (>.5). Did you observe a similar parttern?

**Comment by sowmaster** [comment]
I had to make few debugging on the data processings but finally were able to run code. I am just using the provided domain weights at the moment so didn't get to make that observation yet. Were you able to run it on multi GPU? Thanks!

**Comment by punisher1k** [comment]
Yes, just run the code as it is and accelerate will take care of that

**Comment by melisilaydabal** [comment]
Hi @sowmaster! Would you be kind enough to share your debugged scripts for preprocessing. I also can't run this code as is, not on multi-GPU either. Many thanks!
